core_test.go

```diff
@@ -261,6 +261,144 @@ func TestCore_GetByRelationship_update(t *testing.T) {
 	}
 }
 
+func TestCore_GetByRelationship_many_to_many(t *testing.T) {
+	var (
```

Comment:
Test coverage looks good, but some comments or messages about why the expected values are the intended results, and what "might" be wrong if they get an unexpected value

Reply:
I love this suggestion. I'll add it